Gifski/EditVideoViewController.swift

```diff
@@ -425,7 +453,23 @@ final class EditVideoViewController: NSViewController {
 		Defaults.observe(.loopGif) { [weak self] in
 			self?.playerViewController.loopPlayback = $0.newValue
 		}
-			.tieToLifetime(of: self)
+		.tieToLifetime(of: self)
+
+		Defaults.observe(.bounceGif) { [weak self] in
+			self?.playerViewController.bouncePlayback = $0.newValue
+			self?.estimateFileSize()
+		}
+		.tieToLifetime(of: self)
+
+		playerRateObserver = playerViewController.playerView.player?.observe(\.rate, options: [.new]) { [weak self] _, change in
+			guard change.newValue == -1 else {
+				return
+			}
+
+			SSApp.runOnce(identifier: "lowKeyframeRateWarning") {
```

Comment:
I just realized that this will only show the warning if the first video we shows that plays in reverse has low key frame rate, otherwise `showKeyframeRateWarningIfNeeded()` will never be hit. So in the likely scenario, most users will not see the warning even if they eventually load a video with a low key frame rate.

Reply:
Commit becc38f changes the behavior so the warning is shown once per-conversion that has a low keyframe rate. And includes the option to suppress the warning for all future conversions:
<img width="917" alt="alert-with-suppression" src="https://user-images.githubusercontent.com/915431/115976443-146c1c80-a523-11eb-9753-75e2fdd80a26.png">